.devcontainer.json

```diff
@@ -1,5 +1,10 @@
 {
   "name": "terraform",
   "extensions": ["hashicorp.terraform"],
-  "postCreateCommand": "/bin/bash -c 'curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add - && sudo apt-add-repository \"deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main\" && sudo apt-get update && sudo apt-get install terraform'"
+  "postCreateCommand": "/bin/bash -c 'curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add - && sudo apt-add-repository \"deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main\" && sudo apt-get update && sudo apt-get install terraform'",
+  "hostRequirements": {
+     "cpus": 4,
+     "memory": "8gb",
```

Comment:
too much?

Reply:
The immediately lower machine type ends up killing `go build` with out–of–memory.